analysis/src/HistoManager.cxx

```diff
@@ -204,6 +212,8 @@ void HistoManager::saveHistos(TFile* outF,std::string folder) {
         dir->cd();
     }
 
+    event_h->Write();
```

Comment:
Buggy if you have more than  one instance of histograms writing on the same output file